mptt/managers.py

```diff
@@ -616,6 +616,7 @@ def move_node(self, node, target, position='last-child'):
         move the node yourself by setting node.parent.
         """
         self._move_node(node, target, position=position)
+        node.save()
```

Comment:
The commit removes the `parent` updating from `_move_node`, that's why a separate `save()` is necessary.

The reason for this is that `parent` should be updated at the same time as other fields, not at the same time as the MPTT fields if we want to support unique constraints containing parent-related data.